### DISCUSSION TITLE
gpu stats

### DISCUSSION THREAD

**Discussion by kirsch33** [blakeblackshear/frigate#6654] [Ask A Question]
Under the system tab for GPU stats, the memory % listed, is that system memory (RAM) or GPU memory that is being listed?

I ask because I keep track of my GPU usage over time with Grafana/InfluxDB and the memory % in this panel never is remotely close to what I see from my Grafana dashboard. So the only way this makes sense to me is if this is in fact RAM? I suppose since Grafana is taking the mean value over time that could dilute the overall usage if every single time I happen to look in Frigate and the value is higher or something like that but it seems unlikely. Frigate is often reporting 30%+ memory usage and according to Grafana my GPU usage hasnt been above 10% in weeks

**Comment by NickM-27** [comment]
This is the GPU memory usage, it is taken directly from radeontop over Nvidia smi
